core/embed/rust/src/ui/util.rs

```diff
@@ -131,6 +131,28 @@ pub fn char_to_string(ch: char) -> ShortString {
     s
 }
 
+/// Returns the first two lines of text to be fit on a given length.
+/// When the text is too long to fit in two lines, the rest is just kept as part of the second line.
+/// This assumes no lines are longer than 50 chars (ShortString limit)
+pub fn split_two_lines(
```

Comment:
this could easily return `(&str, &str)`  (and then at call site you'd stuff the `shape::Text` inside the tstring.map() closure)

Reply:
> this could easily return `(&str, &str)` (and then at call site you'd stuff the `shape::Text` inside the tstring.map() closure)

Done: https://github.com/trezor/trezor-firmware/pull/4226/commits/fcfc1a2689d93d95d4e1a7385401c26a148f0844